build.gradle

```diff
@@ -62,6 +62,7 @@ dependencies {
     testImplementation 'org.slf4j:slf4j-simple:2.0.5'
     testImplementation 'com.fasterxml.jackson.core:jackson-databind:2.14.2'
     testImplementation 'org.junit.jupiter:junit-jupiter-api:5.8.0'
+    testImplementation 'org.junit.platform:junit-platform-launcher:1.8.0'
```

Comment:
This dependency is needed for using the `TestExecutionListener` and registering it.